x-pack/plugins/endpoint/common/types.ts

```diff
@@ -25,35 +25,39 @@ export type ImmutableObject<T> = { readonly [K in keyof T]: Immutable<T[K]> };
 export class EndpointAppConstants {
   static ALERT_INDEX_NAME = 'my-index';
   static ENDPOINT_INDEX_NAME = 'endpoint-agent*';
-}
-
-export interface AlertResultList {
+  static EVENT_INDEX_NAME = 'endpoint-events-*';
```

Comment:
This isn't really a comment specific to your PR, but I notice that we have lots of types in 'common.ts', and then we have run time code here in 'types.ts'. This seems counterintuitive to me. Also, we have a variety of values in our `config.ts`, but these values, the ES index patterns, seem like the only values that should be configurable to me. I'm making assumptions here since I don't totally understand. But I'm curious about the reasoning.

Reply:
I think the question about index configuration is what is the cascading change if some changes the pattern of the index. Till we can answer that question, I think we should go with convention for now, we already have configuration so we can always move in that direction.

Reply:
@oatkiller When we start to expand out Resolver's implementations outside endpoint we should consider a way to pass some kind of value that allows us to specify the index patterns.